.github/workflows/trivy.yml

```diff
@@ -1,11 +1,21 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2024 Intel Corporation.
+
 name: Trivy scan
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '45 8 * * *'
```

Comment:
Does the scheduled run add any value if we are (correctly I will add) running this on both PRs and landings?

Reply:
There are three usecases
- `PR` - to inform the PR's author about new problems delivered together with the PR - this scan does not add anything to Security tab
- `push` reports new issues introduced by PR to the Security tab if not resolved by the author
- `schedule` reports new issues to the Security tab detected mainly due to new issues reported by CVE databases
There is an overlap between `schedule` and `push` but the `schedule` does not need to wait for any PR to land.